modules/internal/ChapelRange.chpl

```diff
@@ -359,29 +382,29 @@ module ChapelRange {
 
   // Range builders for low bounded ranges
   proc chpl_build_low_bounded_range(low: integral)
-    return new range(low.type, BoundedRangeType.boundedLow, _low=low);
+    return new range(low=low);
   proc chpl_build_low_bounded_range(low: enum)
-    return new range(low.type, BoundedRangeType.boundedLow, _low=low);
+    return new range(low=low);
   proc chpl_build_low_bounded_range(low: bool)
-    return new range(low.type, BoundedRangeType.boundedLow, _low=low);
+    return new range(low=low);
   proc chpl_build_low_bounded_range(low) {
-    compilerError("Bound of 'low..' must be an integer");
+    compilerError("Ranges defined using bounds of type '" + low.type:string + "' are not currently supported");
```

Comment:
Should this say "currently"?  That might imply that we intend to support a type when we don't actually intend to

Reply:
It's used in other places as well, including prior to this line

Reply:
I used that qualifier in order to not rule out cases in the future if a user hit this and wanted to advocate for a specific pattern.  For example, `bigint` is a case where we do intend to support them (but don't yet), and we've talked about `real` being another that we've considered from time to time over the years.  Once we have interfaces, I could imagine users creating their own `rangeable` types as well, such that object types that are int-like could be used.  I don't interpret "currently" as a promise that we will someday (like "yet" would).